src/formatter.rs

```diff
@@ -291,22 +292,22 @@ mod tests {
             formatter.write_str("\n").unwrap();
         }
         insta::assert_snapshot!(String::from_utf8(output).unwrap(), @r###"
-        [30mblack[0m
-        [31mred[0m
-        [32mgreen[0m
-        [33myellow[0m
-        [34mblue[0m
-        [35mmagenta[0m
-        [36mcyan[0m
-        [37mwhite[0m
-        [1;30mbright black[0m
-        [1;31mbright red[0m
-        [1;32mbright green[0m
-        [1;33mbright yellow[0m
-        [1;34mbright blue[0m
-        [1;35mbright magenta[0m
-        [1;36mbright cyan[0m
-        [1;37mbright white[0m
+        [38;5;0mblack[39m
```

Comment:
Do you know why this test changed? What is the difference between `1;30` and `38;5;1`? Why did `0m` become `39m`?

I might look up the answer myself, but I mainly wanted to double-check that this is expected.